src/conventional/balloon_sonde_json2ioda.py

```diff
@@ -154,8 +154,10 @@ def main(args):
         longitude = [file['observations'][ii]['longitude'] for ii in range(len(file['observations']))]
         stationIdentification = [file['observations'][ii]['mission_name'] for ii in range(len(file['observations']))]
         pressure = [file['observations'][ii]['pressure'] for ii in range(len(file['observations']))]
-        windEastward = [file['observations'][ii]['speed_x'] for ii in range(len(file['observations']))]
-        windNorthward = [file['observations'][ii]['speed_y'] for ii in range(len(file['observations']))]
+        #### NOTE: THESE 2 ARE TEMPORARILY REVERSED UNTIL DATA LABELING CONFIRMED
```

Comment:
we should really add the check on file creation date, or software version to trigger this guy

thanks for acting so quickly @smaticka